docs/rst/introduction/metapackages.rst

```diff
@@ -31,8 +31,8 @@ Vulcanexus Tools
 ----------------
 
 **Vulcanexus Tools** is focused on helping developers with the introspection and debugging of their system.
-Thus, the `ROS 2 Monitor <https://fast-dds-monitor.readthedocs.io/en/latest/>`_ and the `ROS 2 Statistics Backend <https://fast-dds-statistics-backend.readthedocs.io/en/latest/>`_ are included in order to monitor and diagnose the *ROS 2* communication network.
-Also, `ROS 2 Shapes Demo <https://eprosima-shapes-demo.readthedocs.io/en/latest/index.html>`_ is provided as a simple demo to help testing that the communication is successfully established in the system.
+Thus, the :ref:`ROS 2 Monitor <vulcanexus_monitor>` and the `ROS 2 Statistics Backend <https://fast-dds-statistics-backend.readthedocs.io/en/latest/>`_ are included in order to monitor and diagnose the *ROS 2* communication network.
```

Comment:
Why the statistics backend does not have its own page?

```suggestion
Thus, the :ref:`ROS 2 Monitor <vulcanexus_monitor>` and the :ref:`ROS 2 Statistics Backend <vulcanexus_backend>` are included in order to monitor and diagnose the *ROS 2* communication network.
```

Reply:
Using `Fast DDS Statistics Backend` instead.